core/src/main/java/com/linecorp/armeria/common/util/ShutdownHooks.java

```diff
@@ -42,6 +43,8 @@ public final class ShutdownHooks {
     private static final Map<AutoCloseable, Queue<Runnable>> autoCloseableOnShutdownTasks =
```

Comment:
Lets' add `@GuardedBy("reentrantLock")` as well.

Reply:
thanks for pointing out my laziness
I also checked and found that only that place had to be fixed
p.s. I realized import code was wrong, fixed it!😆
[32ad6aa](https://github.com/line/armeria/pull/4551/commits/32ad6aa75cbd060e7cd7385c75c8a42fa6344fa8)